src/java.base/share/classes/java/security/KDFParameters.java

```diff
@@ -38,8 +38,12 @@
  * {@link javax.crypto.KDF#getInstance(String, KDFParameters) KDF.getInstance}
  * methods. The {@code getInstance} method returns a {@code KDF}. The
  * {@code KDF} that is returned is initialized with the specified parameters.
+ * <p>
+ * The {@code KDFParameters} used for initialization can be retrieved via
+ * {@link javax.crypto.KDF#getParameters()}.
```

Comment:
Since you add this line here right after the "initialized with" line, there should be some words about the difference between the user-provided params and actual params.

Reply:
The new paragraph is a little too long. Something like "Users provide request params ti getInstance, and get actual params in getParams(). See KDF.getParams for details" is enough.